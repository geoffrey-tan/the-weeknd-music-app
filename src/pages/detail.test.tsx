import { render } from '@testing-library/react'
import { MemoryRouter } from 'react-router-dom'

import DetailPage from './detail'

describe('App', () => {
  it('renders', () => {
    const { container } = render(
      <MemoryRouter
        initialEntries={['/f05567cc-6ed3-40e0-bad1-7812478eecbe/so-far-gone']}
      >
        <DetailPage />
      </MemoryRouter>
    )

    expect(container).toMatchSnapshot()
  })
})
