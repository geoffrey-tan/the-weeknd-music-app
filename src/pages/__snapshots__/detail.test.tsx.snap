// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailPage renders 1`] = `
<div>
  <a
    class="sc-bczRLJ kfAyjk"
    href="/"
  >
    <svg
      aria-label="Previous"
      class="StyledIcon-sc-ofa7kd-0 kKhdEN"
      viewBox="0 0 24 24"
    >
      <path
        d="M17 2 7 12l10 10"
        fill="none"
        stroke="#000"
        stroke-width="2"
      />
    </svg>
    Back
  </a>
  <div
    class="sc-dkzDqf jKeDgZ"
  >
    <div
      class="StyledBox-sc-13pk1d4-0 ffyxzI"
    />
  </div>
</div>
`;
